app/views/shared/steps/_broken-links-summary.html.erb

```diff
@@ -1,5 +1,8 @@
 <% if step.link_report? %>
-  <span class="govuk-tag govuk-tag--<%= step.broken_links? ? 'warning' : 'success' %>">
-    <%= @step_by_step_page_presenter.broken_link_status_text(step) %>
-  </span>
+  <div class="step-broken-links__summary">
+    <span class="app-broken-link-status app-broken-link-status--<%= step.broken_links? ? 'fail' : 'pass' %>">
+      <%= @step_by_step_page_presenter.broken_link_status_icon(step) %>
```

Comment:
It feels strange to be calling `@step_by_step_page_presenter` here as we're in an individual step.

Reply:
As the icon is static, perhaps instead this can be:

```
<% if step.broken_links? %>
  render "cross.svg"
<%  else %> 
  render "tick.svg"
<% end %>
```

I think for this to work, `cross.svg` and `tick.svg` need to be moved somewhere accessible by the view.

Reply:
I've got this working:

```
<% if step.link_report? %>
  <div class="step-broken-links__summary">
    <% if step.broken_links? %>
      <span class="app-broken-link-status app-broken-link-status--fail">
        <%= render "svgs/cross.svg" %>
        <%= pluralize(step.broken_links.count, "broken link") %> found
      </span>
    <% else %>
      <span class="app-broken-link-status app-broken-link-status--pass">
        <%= render "svgs/tick.svg" %>
        No broken links
      </span>
    <% end %>
  </div>
<% end %>
```

The svg files are in `app/views/svgs` they are being rendered as partials, so they need a `_` prepended to the file name, e.g. `_cross.svg`